packages/orbit-components/src/CarrierLogo/index.tsx

```diff
@@ -52,22 +53,32 @@ const getURLSizes = ({ size }) => {
 };
 
 const StyledImage = styled.img.attrs<StyledProps>(
-  ({ carrierType = "airline", carriersLength, size, code }) => {
+  ({ carrierType = "airline", carriersLength, size, code, inlineStacked }) => {
     const urlSizes =
-      carriersLength > 1 ? getURLSizes({ size: SIZE_OPTIONS.SMALL }) : getURLSizes({ size });
+      carriersLength > 1 && !inlineStacked
+        ? getURLSizes({ size: SIZE_OPTIONS.SMALL })
+        : getURLSizes({ size });
+
     return {
       src: `${BASE_URL}/airlines/${urlSizes.base}x${urlSizes.base}/${code}.png?default=${carrierType}.png`,
       srcSet: `${BASE_URL}/airlines/${urlSizes.retina}x${urlSizes.retina}/${code}.png?default=${carrierType}.png 2x`,
     };
   },
 )`
-  ${({ theme, rounded }: StyledProps) => css`
+  ${({ theme, rounded, inlineStacked }: StyledProps) => css`
     background-color: ${theme.orbit.backgroundCarrierLogo};
     border-radius: ${rounded ? theme.orbit.borderRadiusCircle : theme.orbit.borderRadiusNormal};
     height: ${getCarrierLogoSize};
     width: ${getCarrierLogoSize};
+    border: ${inlineStacked && `1px solid ${theme.orbit.paletteWhite}`};
+    box-shadow: ${inlineStacked && `${theme.orbit.boxShadowFixed}, ${theme.orbit.boxShadowRaised}`};
+
+    &:not(:first-child) {
+      margin-left: ${inlineStacked && `calc(-1 * ${theme.orbit.spaceXSmall})`};
```

Comment:
use left utility function here from RTL utilities

Reply:
Please check if I've done it correctly 😅

Reply:
added suggestion below ;)